data.md

```diff
@@ -457,8 +470,8 @@ We are using the polymorphic `Map` sort for these word maps.
 ```{.k .uiuck .rvk}
     syntax Array ::= Array "[" Int ":=" WordStack "]" [function]
  // --------------------------------------------------------
-    rule WM[ N := .WordStack ] => WM
-    rule WM[ N := W : WS     ] => (WM[chop(N) <- W])[chop(N) +Int 1 := WS]
+    rule WM::Array[ N := .WordStack ] => WM
+    rule WM::Array[ N := W : WS     ] => (WM[chop(N) <- W])[chop(N) +Int 1 := WS]
```

Comment:
This should use `N +Int 1` instead of `chop(N)+Int 1`. The number in `[chop(N) <- W]` is already always getting chopped. Having an extra call on the right hand side will just complicated proving by giving us formulas like `chop(chop(chop(X)+1)+1)+1` instead of `X+1+1+1`.